lib/iris/tests/unit/fileformats/pp_load_rules/test__epoch_date_hours.py

```diff
@@ -0,0 +1,134 @@
+# (C) British Crown Copyright 2014 - 2017, Met Office
```

Comment:
Need to change these years. And it looks you need to update the dates on the other files as well

Reply:
Ok, will fix.

It's annoying that this test has stopped working in Travis, not sure when that happened but several headers are now out-of-date.
There was also talk of ditching the file headers, or at least the dates, but discussions with internal legal team are not complete AFAIK.
#3272 Attempts to fix ..

Reply:
Can now wait for #3272, and rebase when that is in.